jobs/datadog-cc-worker/templates/process.yaml.erb

```diff
@@ -0,0 +1,6 @@
+init_config:
+
+instances:
+  - name: cc api worker
```

Comment:
yes, ofc. 

I will change this and all the other things you mention. Thx for spotting them.